### PR TITLE
fix modification of same abtest not applying

### PR DESCRIPTION
**Pull request by yarneo**

### CODE REVIEW

File: Mixpanel/MPVariant.m [hunk 1]
```diff
@@ -241,7 +241,7 @@ - (void)restart {
 
 - (BOOL)isEqualToVariant:(MPVariant *)variant
 {
-    return self.ID == variant.ID;
+    return self.ID == variant.ID && self.actions == variant.actions;
```
Comment (pchien) [inline Mixpanel/MPVariant.m]: this needs to be `isEqual` instead of `==`

**Comment by jaredmixpanel** [comment]
@yarneo just want to confirm, this means that modifications to existing variants will get applied to devices that had already received that variant, instead of always using the cached one, correct?

**Comment by pchien** [comment]
Yep. This makes modifying A/B tests that are already running change for users who already have the A/B test.

**Comment by peymano** [comment]
@jaredmixpanel Does the Android SDK implement this behavior too?

**Pull request merged**